tests/frontier/opcodes/test_calldataload.py

```diff
@@ -0,0 +1,140 @@
+"""
+abstract: Test `CALLDATALOAD`
+    Test the `CALLDATALOAD` opcodes.
+
+"""
+
+import pytest
+
+from ethereum_test_forks import Frontier, Homestead
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    "sstore_test_opcode,calldata_opcode,validate_length,expected_storage",
+    [
+        (
+            Bytecode(Op.SSTORE(key=0x0, value=Op.CALLDATALOAD(offset=0x0))),
+            Bytecode(Op.MSTORE8(offset=0x0, value=0x25) + Op.MSTORE8(offset=0x1, value=0x60)),
+            0x02,
+            (
+                Account(
+                    storage={
+                        0x00: 0x2560000000000000000000000000000000000000000000000000000000000000
+                    }
+                )
+            ),
+        ),
+        (
+            Bytecode(Op.SSTORE(key=0x0, value=Op.CALLDATALOAD(offset=0x01))),
+            Bytecode(
+                Op.MSTORE(
+                    offset=0x0,
+                    value=0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
+                )
+                + Op.MSTORE8(offset=0x20, value=0x23),
+            ),
+            0x21,
+            (
+                Account(
+                    storage={
```

Comment:
This seems to be expected calldataload memory. 
Can omit Account, storage defenition on each parameter